mod_bot/main.py

```diff
@@ -56,6 +56,18 @@ def add_minutes(hstart, mstart, mn):
     print(conf)
     is_connected = True
 
+
+def clean_spaces(command):
+    return clean_char(command, ' ')
```

Comment:
this will not work as all the whitespace characters will be removed, leaving an empty string, and thus "passing" the filter test

Reply:
it will because I am not testing against a string but a list. I will put some test so it is explicit

Reply:
you were right sorry, I didn't understand what you said :D